Applications/web_portal/web_app/library/source/pages/account_page/risk_page/risk_parameters_view.tsx

```diff
@@ -8,13 +12,317 @@ interface Properties {
 
   /** Used to lookup currency names and symbols. */
   currencyDatabase: Nexus.CurrencyDatabase;
+
+  /** Called on an update of the currency. */
```

Comment:
None of these props can be here. When currency or buying power or anything changes, instead of propagating a signal, you update the model with the new values.